release_notes.md

```diff
@@ -8,4 +8,5 @@
 - Update WebJobsScriptHostService to remove hardcoded sleep during application shut down (#9520)
 - Update PowerShell 7.2 Worker Version to [4.0.2974](https://github.com/Azure/azure-functions-powershell-worker/releases/tag/v4.0.2974)
 - Update PowerShell 7.0 Worker Version to [4.0.2973](https://github.com/Azure/azure-functions-powershell-worker/releases/tag/v4.0.2973)
-- Add support for standalone executable (ie: `dotnet build --standalone`) for out-of-proc workers in Linux Consumption.
+- Limit dotnet-isolated specialization to 64 bit host process (#9548).
```

Comment:
nit: Limit dotnet-isolated use of new native placeholders to 64bit host processes only.

Specialization happens regardless if they use native placeholders or base placeholders.